### PR TITLE
Fix popover position

### PR DESCRIPTION
**Pull request by KaroMourad**
Fix: 
 - popover position on chart scroll
 - popover position on zoom mode (related by chart dimensions)
 
 Add: 
 - reusable ChartPopover component (bounded by container if needed)
 
 In the future ChartPopover component can renamed by UIPopover and use it when it needed

### CODE REVIEW

File: aim/web/ui_v2/src/components/ChartPanel/ChartPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Popover, PopoverPosition } from '@material-ui/core';
+import { isEqual } from 'lodash-es';
+
+import useStyles from './ChartPopoverStyle';
+import { IChartPopover } from '../../types/components/ChartPanel/ChartPopover';
```
Comment (arsengit) [inline aim/web/ui_v2/src/components/ChartPanel/ChartPopover.tsx:6]: lets use absolute path here

File: aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+export default makeStyles((theme) => ({
+  popover: {
+    pointerEvents: 'none',
+  },
+  popoverContent: {
+    margin: '1em',
+    width: 250,
+    height: 60,
+  },
+}));
```
Comment (arsengit) [inline aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts:12]: I think we should start writing styles in scss files

File: aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts [hunk 3]
```diff
@@ -8,6 +8,7 @@ import { CurveEnum } from 'utils/d3';
 import { SmoothingAlgorithmEnum } from 'utils/smoothingData';
 import { IMetric } from './metricModel';
 import { IRun } from './runModel';
+import { IActivePoint } from '../../../utils/d3/drawHoverAttributes';
```
Comment (arsengit) [inline aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts:11]: About absolute path also here

**Review by arsengit — APPROVED** [review]

**Review by VkoHov — APPROVED** [review]

**Review by roubkar — APPROVED** [review]

File: aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts:12]: did you refactor all the styles?

Comment (arsengit) [reply aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts:12]: No, but the new styles I writing in scss files.

Comment (KaroMourad) [reply aim/web/ui_v2/src/components/ChartPanel/ChartPopoverStyle.ts:12]: ok i will fix this, but for all project we can refactor it in one task

File: aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts (hunk 3 above)
Comment (KaroMourad) [reply aim/web/ui_v2/src/types/services/models/metrics/metricsAppModel.d.ts:11]: In some cases this absolute path does not works well and you cant write all paths absolute, i think we can rethink about this

**Pull request merged**